cves/CVE-2013-0870.yml

```diff
@@ -348,7 +384,14 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    In my opinion, I think this was either a design oversight or a poor job
+    of code review. Designs could've initially planned for this thread check
+    to only apply to a specific type, but failed to realize it is necessary
+    for all types. Or, the original developer could have misplaced this
+    check from the jump, and lack of testing/code review didn't allow this
+    vulnerability to caught before it was released. Thankfully it was caught
```

Comment:
Can you give a reason why you think it was found so fast? Others in this software took years to find